src/components/viz/ReviewCase.vue

```diff
@@ -154,7 +150,7 @@
                 <v-icon class="good-coverage" v-on="on" top color="green"
                         @mouseover="onHover = true" @mouseleave="onHover = false" @click="">check_circle</v-icon>
               </template>
-              <span>Median coverage is above expected {{isExomeText}} coverage threshold of {{minCutoff}}X</span>
+              <span>Median coverage is above expected coverage threshold of {{minCutoff}}X</span>
```

Comment:
Hi @WilliamRichards2017 , 
If the input box is cleared and you do a mouseover on the "check_circle" icon. Should it show "Median coverage is above expected coverage threshold of 0 X"? 
<img width="715" alt="Screen Shot 2020-03-06 at 4 25 09 PM" src="https://user-images.githubusercontent.com/16284713/76132355-6c665d80-5fc7-11ea-9b85-7f7040262f41.png">

Also if the value is set at 10, the min threshold indicator is showing outside the graph. 
<img width="709" alt="Screen Shot 2020-03-06 at 4 25 21 PM" src="https://user-images.githubusercontent.com/16284713/76132448-f3b3d100-5fc7-11ea-9ad4-24478e215ef3.png">